sui/src/config/mod.rs

```diff
@@ -466,8 +508,9 @@ impl<C> DerefMut for PersistedConfig<C> {
     }
 }
 
-/// Make a default Narwhal-compatible committee.
-pub fn make_default_narwhal_committee(
+/// Make a default Narwhal-compatible committee running locally
+/// Ports are overridden by next available port id.
```

Comment:
To be more specific, we should say that it is designed either for tests or running multiple validators in different threads locally.   Basically this should not be used for production/deployed scenarios.